packages/grid/src/InternalGrid.tsx

```diff
@@ -0,0 +1,312 @@
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import React, {
+  useState,
+  useRef,
+  useEffect,
+  useCallback,
+  MutableRefObject,
+  useMemo,
+} from 'react';
+import {
+  Grid as VirtualisedGrid,
+  CellMeasurer,
+  ScrollPosition,
+  GridCellProps,
+} from 'react-virtualized';
+import { setIn } from 'utilitify';
+import {
+  Body,
+  BodyCell,
+  BodyCellOffset,
+  BodyCellContent,
+  ExpandButtonWrapper,
+  Wrapper,
+  TotalBlock,
+  TotalCellContent,
+  TotalCell,
+  ShiftInsteadButton,
+  TotalsShift,
+} from './styled';
+import { AddIcon, RemoveIcon } from './icons';
+import { searchLastVisible } from './utils';
+
+import { HeaderWrapper } from './HeaderWrapper';
+import {
+  IColumn, IInternalGrid,
+} from './interfaces';
+
+
+const InternalGrid = ({
```

Comment:
use this type description method for the component like:
`const InternalGrid: React.FC<IInternalGrid> =>`

It's better because you describe here props and return type